src/audio/data.c

```diff
@@ -74,33 +74,38 @@ s32 reverbMultsArr[][NUM_ALLPASS / 3] = {
- * - reverbIndex    (Advanced parameter used to tune the incoming output of every third filter)
+ * - useLightSettings (Reduce some runtime configurability options in favor of a slight speed boost during processing; Light configurability settings are found in synthesis.h)
+ * - downsampleRate   (Higher values exponentially reduce the number of input samples to process, improving perfomance at cost of quality)
+ * - isMono           (Only process reverb on the left channel and share it with the right channel, improving performance at cost of quality)
+ * - filterCount      (Number of filters to process data with; in general, more filters means higher quality at the cost of performance demand; always 3 with light settings)
+ * 
+ * - windowSize       (Size of circular reverb buffer; higher values work better for a more open soundscape, lower is better for a more compact sound)
+ * - gain             (Amount of audio retransmitted into the circular reverb buffer, emulating decay; higher values represent a lengthier decay period)
+ * - gainIndex        (Advanced parameter; used to tune the outputs of every first two of three filters; overridden when using light settings)
+ * - reverbIndex      (Advanced parameter; used to tune the incoming output of every third filter; overridden when using light settings)
  * 
- * - *delaysL       (Array of variable audio buffer sizes / delays for each respective filter [left channel])
- * - *delaysR       (Array of variable audio buffer sizes / delays for each respective filter [right channel])
- * - *reverbMultsL  (Array of multipliers applied to the final output of each group of 3 filters [left channel])
- * - *reverbMultsR  (Array of multipliers applied to the final output of each group of 3 filters [right channel])
+ * - *delaysL         (Advanced parameter; array of variable audio buffer sizes / delays for each respective filter [left channel]; overridden when using light settings)
+ * - *delaysR         (Advanced parameter; array of variable audio buffer sizes / delays for each respective filter [right channel]; overridden when using light settings)
+ * - *reverbMultsL    (Advanced parameter; array of multipliers applied to the final output of each group of 3 filters [left channel])
+ * - *reverbMultsR    (Advanced parameter; array of multipliers applied to the final output of each group of 3 filters [right channel])
  * 
- * NOTE: First entry will always be used by default when not using the level commands to specify a preset.
+ * NOTE: The first entry will always be used by default when not using the level commands to specify a preset.
  * Please reference the HackerSM64 Wiki for more descriptive documentation of these parameters and usage of BETTER_REVERB in general.
  */
 struct BetterReverbSettings gBetterReverbSettings[] = {
     { /* 0 */
-        -1, FALSE, NUM_ALLPASS, -1, -1, 0x00, 0x00, // Vanilla Reverb
+        FALSE, -1, FALSE, NUM_ALLPASS, // Vanilla Reverb
```

Comment:
these structs are getting pretty big, I think it might be worth adding dot syntax to them

Reply:
On one hand, I'm conflicted because that'll take a ton of space.

But on the other hand, these arguments are so different that you're probably right.